code/modules/mob/grab/scp/grab_049.dm

```diff
@@ -12,20 +12,18 @@
 	type_name = GRAB_PLAGUE_DOCTOR
 	icon = 'icons/mob/screen1.dmi'
 
-	can_absorb = 1
-	point_blank_mult = 1
-	ladder_carry = 1
-	force_danger = 1
-	can_grab_self = 0
-	stop_move = 1
-	reverse_facing = 0
-	same_tile = 0
-
-	/// When TRUE - blocks DoEffects() proc from continuing
+	ladder_carry = TRUE
+	force_danger = TRUE
+	stop_move = TRUE
+	can_grab_self = FALSE
+
+	action_cooldown = 10 SECONDS
+
+	/// When TRUE - blocks quick_stabs() proc from continuing
 	var/stop_effects = FALSE
 
-// This causes the user to temporarily stun and weaken the target, dealing pain and slight brute damage
-/datum/grab/plague_doctor/proc/AttemptCure(obj/item/grab/normal/G)
```

Comment:
bruh

Reply:
Dont rename the proc, we love PascalCase now

Reply:
fuck pascalcase

Reply:
Ok then I'm closing this